Makefile

```diff
@@ -241,6 +243,82 @@ catalog-deploy:
 	sed -e 's~<IMG>~$(CATALOG_IMG)~' ./config/samples/catalog/catalog.yaml | kubectl apply -f -
 
 # Deploy the sample FlowCollector CR
-.PHONY: create-sample
-create-sample:
-	sed -e 's~:main~:v$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl apply -f -
+.PHONY: deploy-sample-cr
+deploy-sample-cr:
+	sed -e 's~:main~:$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl apply -f - || true
+
+# Undeploy the sample FlowCollector CR
+.PHONY: undeploy-sample-cr
+undeploy-sample-cr:
+	sed -e 's~:main~:$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl --ignore-not-found=true delete -f -  || true
+
+##@ Development
+
+.PHONY: deploy-loki
+deploy-loki: ## Deploy loki.
+	@echo "### Deploying loki"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl create -f -  || true
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml	 | kubectl create -f -  || true
```

Comment:
Not sure I want `|| true` here.

Reply:
@stleerh this is for development ... I do not want to fail if this is already deployed .... just move on ... in production ... yes you are totally correct but for dev ??? the logic is let's go on and do the best we can ... if something exists let's use it and do not fail ... same with double delete.